.coveragerc

```diff
@@ -1196,6 +1196,9 @@ omit =
     homeassistant/components/whois/sensor.py
     homeassistant/components/wiffi/*
     homeassistant/components/wirelesstag/*
+    homeassistant/components/wiz_light/__init__.py
+    homeassistant/components/wiz_light/const.py
+    homeassistant/components/wiz_light/light.py
```

Comment:
It is not compulsory, but since new integrations can take a while to get merged, I suggest that you take the opportunity to move to 100% coverage whilst the reviews are in progress.
```suggestion
```